ding/ding.py

```diff
@@ -82,10 +82,15 @@ def print_time(seconds):
     """Print countdown for `seconds`"""
     while seconds > 0:
         start = time.time()
-        os.system('cls' if os.name == 'nt' else 'clear') # accommodate Windows
-        print(datetime.timedelta(seconds=seconds))
+        if os.name == 'nt':
+            os.system('cls')
+            print(datetime.timedelta(seconds=seconds))
+        else:
+            print(datetime.timedelta(seconds=seconds), end='')
```

Comment:
Why can't we just use `end='\r'` and avoid line 93 altogether? It seems to work for me.